projects/example-app-cypress/integration/round-trip.spec.ts

```diff
@@ -1,14 +1,25 @@
 context('Full round trip', () => {
   before(() => {
+    (cy as any).eyesOpen({
+      appName: 'books_app',
+      testName: 'round-trip',
+      browser: { width: 800, height: 600 },
+    });
     window.indexedDB.deleteDatabase('books_app');
     cy.visit('/');
   });
+  after(() => {
```

Comment:
```suggestion

  after(() => {
```

Reply:
Hey Brandon, Where is the root cypress.json you want removed?

Reply:
The cypress.json file is in the root of your repo folder. Its at the same level as the `angular.json` file.